src/jit/gentree.h

```diff
@@ -574,11 +574,21 @@ struct GenTree
         return isContained() && IsCnsIntOrI() && !isUsedFromSpillTemp();
     }
 
+#if FEATURE_X87_DOUBLES
```

Comment:
I had done it since there is no concept of a 'float' with regards to x87 FPU, instead everything is 80 bits.

I can certainly drop all the `#if FEATURE_X87_DOUBLES` and either:
* Just use the new code everywhere, or
* Switch to a different define

If a different approach would be more ideal, please let me know.